### PR TITLE
Proper versioning import

### LINKED ISSUE #59: Some libraries are not detected
It seems some libraries are not correctly detected.

```text
Core packages
-------------
ansys.mapdl.core    : 0.64.dev0
numpy               : 1.23.5
appdirs             : 1.4.4
scipy               : 1.9.3
grpc                : Package not found
ansys.api.mapdl.v0  : Package not found
ansys.mapdl.reader  : 0.52.4
google.protobuf     : Package not found
pexpect             : 4.8.0
```

Link: https://github.com/pyansys/pymapdl/actions/runs/3733984882/jobs/6335972283#step:7:151

### PR DESCRIPTION
**Pull request by RobPasMue**
Closes #59

### CODE REVIEW

**Comment by RobPasMue** [comment]
Looks like these changes will solve your issues @germa89 

image

**Comment by RobPasMue** [comment]
Also, adding additional deps:

image

**Comment by RobPasMue** [comment]
If PR is merged, a new release will be required.

**Review by MaxJPRey — APPROVED** [review]
LGTM.

File: src/ansys/tools/report/report.py [hunk 1]
```diff
@@ -80,15 +80,15 @@ def __init__(
...
             "numpy",
-            "pyvista",
-            "appdirs",
-            "tqdm",
             "pyiges",
+            "pyvista",
             "scipy",
-            "grpc",  # grpcio
-            "google.protobuf",  # protobuf library
+            "tqdm",
```
Comment (germa89) [inline src/ansys/tools/report/report.py:91]: Why the order matters??

**Review by germa89 — COMMENTED** [review]
LGTM.

File: src/ansys/tools/report/report.py (hunk 1 above)
Comment (RobPasMue) [reply src/ansys/tools/report/report.py:91]: Alphabetical order is always better than no order 😄

Comment (germa89) [reply src/ansys/tools/report/report.py:91]: Loooll... true!

**Pull request merged**